webpack.config.common.js

```diff
@@ -181,12 +181,8 @@ module.exports = {
                 transform: replaceVersion
             },
             {
-                from: 'src/js/angular/core/templates/modal',
-                to: 'js/angular/core/templates/modal'
-            },
-            {
-                from: 'src/js/angular/templates',
-                to: 'js/angular/templates'
+                from: 'src/js/angular/core/components/export-settings-modal',
+                to: 'js/angular/core/components/export-settings-modal'
```

Comment:
Can you also put this `transform: replaceVersion` at the end! This ensures that a new hash is added in the end of the file name on each change of the template so that the browser to know that there is a change in the file and it should be reloaded.